docs/app/pages/components/components-sections/date-time-picker/date-range-picker/date-range-picker.component.ts

```diff
@@ -78,7 +78,7 @@ export class ComponentsDateRangePickerComponent extends BaseDocumentationSection
         // check if the date contains a hyphen
         const parts = (date.indexOf('—') ? date.split('—') : date.split('-'));
         const startDate = Date.parse(parts[0].trim());
-        const endDate = Date.parse(parts[0].trim());
+        const endDate = Date.parse(parts[1].trim());
```

Comment:
Does the code snippet/plunker not also need updated?

Reply:
Updated now and added as a comment to the original ticket - https://next.plnkr.co/edit/fGOJRoh1RXUfGAAl